### PR TITLE
docker and detecting internet traffic from a buildpack

### PR DESCRIPTION
**Pull request by davidjahn**
Hey, I'm with the CF buildpacks team in NYC. Submitting PR for a blog post about something we recently implemented.

thanks!
David

### CODE REVIEW

File: content/post/network-traffic-monitoring-with-containers.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+---
+
+
+## Software and Promises
+
+When software makes a promise, engineers need a way to verify that the promise is kept. In modern
+software engineering, the process of “promise checking” is performed by using a __continuous integration__ (CI) system:
+
+<div style="text-align:center">
+{{< figure src="/images/network-monitoring/concourse.png" style="text-align:center" >}}
```
Comment (flavorjones) [inline content/post/network-traffic-monitoring-with-containers.md]: I don't think this is the idiomatic way to embed images.

In my post, I used `class="right small"`, for example. There's also `responsive-figure` which does something else entirely.

 @tsaleh or @Osis, can you provide guidance?

**Comment by davidjahn** [comment]
Hey @flavorjones, fixed the issue you pointed out. @tsaleh hopefully this is OK to merge and flip "draft" to false if everything looks good.

**Comment by tsaleh** [comment]
Hey @davidjahn can you drop a link to the post on staging?

**Comment by tsaleh** [comment]
Actually, can you merge this first, so it's on staging, then drop a link?  You can take off the draft bit later.

**Pull request merged**